common/src/main/java/com/microsoft/identity/common/AndroidCommonComponents.java

```diff
@@ -26,27 +26,55 @@
 
 import androidx.annotation.Nullable;
 
-import com.microsoft.identity.common.crypto.AndroidBrokerStorageEncryptionManager;
 import com.microsoft.identity.common.crypto.AndroidAuthSdkStorageEncryptionManager;
+import com.microsoft.identity.common.crypto.AndroidBrokerStorageEncryptionManager;
+import com.microsoft.identity.common.internal.cache.SharedPreferencesFileManager;
 import com.microsoft.identity.common.internal.net.cache.HttpCache;
+import com.microsoft.identity.common.internal.platform.AndroidDeviceMetadata;
+import com.microsoft.identity.common.internal.platform.DevicePopManager;
 import com.microsoft.identity.common.internal.util.ProcessUtil;
+import com.microsoft.identity.common.internal.util.SharedPreferenceLongStorage;
+import com.microsoft.identity.common.java.crypto.IDevicePopManager;
 import com.microsoft.identity.common.java.crypto.IKeyAccessor;
+import com.microsoft.identity.common.java.exception.ClientException;
 import com.microsoft.identity.common.java.interfaces.ICommonComponents;
+import com.microsoft.identity.common.java.platform.Device;
 import com.microsoft.identity.common.java.telemetry.ITelemetryCallback;
+import com.microsoft.identity.common.java.util.ClockSkewManager;
+import com.microsoft.identity.common.java.util.IClockSkewManager;
 import com.microsoft.identity.common.logging.Logger;
 
+import java.io.IOException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+
 import lombok.NonNull;
 
+import static com.microsoft.identity.common.java.exception.ClientException.CERTIFICATE_LOAD_FAILURE;
+import static com.microsoft.identity.common.java.exception.ClientException.IO_ERROR;
+import static com.microsoft.identity.common.java.exception.ClientException.KEYSTORE_NOT_INITIALIZED;
+import static com.microsoft.identity.common.java.exception.ClientException.NO_SUCH_ALGORITHM;
+
 /**
  * Android implementations of platform-dependent components in Common.
  */
 public class AndroidCommonComponents implements ICommonComponents {
```

Comment:
I'm not a big fan of this interface name.  IPlatformComponents or IOperatingSystemComponents makes more sense to me.  ICommonComponents is very generic and i think potentially confusing.

Reply:
Then you get: AndroidPlatformComponents or AndroidOperatingSystemComponents / LinuxPlatformComponents, LinuxOperatingSystemComponents

Reply:
I'll rename it in a separate PR. (Already have 2 PRs on top of this, and renaming it now would end up in a conflict mess).